magda-web-client/src/UI/DataPreviewMap.js

```diff
@@ -70,7 +70,7 @@ class DataPreviewMap extends Component {
         }
     }
 
-    componentWillMount() {
+    UNSAFE_componentWillMount() {
```

Comment:
Looks like as long as you have `getDerivedStateFromProps` lower down you can just delete this altogether?